app/build.gradle

```diff
@@ -107,7 +107,8 @@ dependencies {
     implementation "androidx.core:core-ktx:1.2.0-beta01"
     implementation "androidx.appcompat:appcompat:1.1.0"
     implementation "androidx.fragment:fragment-ktx:1.1.0"
-    implementation "androidx.browser:browser:1.0.0"
+    implementation "androidx.browser:browser:1.2.0"
+    implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
```

Comment:
looks like `browser:1.0.0` had a dependency on `swiperefreshlayout` that we are now missing

Reply:
Is it like... peer dependency?